js/button.js

```diff
@@ -57,7 +57,7 @@
       var $input = this.$element.find('input')
       if ($input.prop('type') == 'radio') {
         if ($input.prop('checked')) changed = false
-        $parent.find('.active').removeClass('active')
+        $parent.find('.active').removeClass('active').find('input').prop('checked', false)
```

Comment:
could you help me about the coding style, should I break the line before ``.find('input')`` ?